Source/Microsoft.Teams.Apps.FAQPlusPlus/Properties/Strings.resx

```diff
@@ -170,7 +170,7 @@
     <comment>Ability to chat directly with end user from SME channel.</comment>
   </data>
   <data name="NotificationCardContent" xml:space="preserve">
-    <value>I've shared your question with the related experts. They'll respond to you directly with a chat message.</value>
+    <value>I've shared your question with the team. An expert will reach out to you soon in separate chat.</value>
```

Comment:
in a separate chat
(missing the article "a")

Reply:
Updated the string.